IDisposableExercise/IDisposableExercise/Person.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.Win32.SafeHandles;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IDisposableExercise
+{
+    /// <summary>
+    /// Represents a person.
+    /// </summary>
+    public class Person : IDisposable
+    {
+        /// <summary>
+        /// Represents the name of a person.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Flag: Has Disponse already been called?
+        /// </summary>
+        public bool disposed = false;
```

Comment:
To protect your private data, this field should be made private and you should use a public read-only property to expose the value of the property. As implemented, a caller could change the disposed state of the class just by updating the field and without calling the Dispose method. (-0.2)